src/skill.cpp

```diff
@@ -221,6 +228,11 @@ void SkillLevel::readBook(int minimumGain, int maximumGain, int maximumLevel)
     practice();
 }
 
+bool SkillLevel::can_train() const
```

Comment:
Left it as non-static because it could be used to implement skill cap and per-skill scaling later on.